internal/cli/atlas/data_lake_describe.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package atlas
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DataLakeDescribeOpts struct {
+	cli.GlobalOpts
+	store store.DataLakeDescriber
+	name  string
+}
+
+func (opts *DataLakeDescribeOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DataLakeDescribeOpts) Run() error {
+	result, err := opts.store.DataLake(opts.ConfigProjectID(), opts.name)
+	if err != nil {
+		return err
+	}
+
+	return json.PrettyPrint(result)
+}
+
+// mongocli atlas datalake(s) describe name --projectId projectId
+func DataLakeDescribeBuilder() *cobra.Command {
+	opts := &DataLakeDescribeOpts{}
+	cmd := &cobra.Command{
+		Use:   "describe <name>",
+		Short: description.DescribeDataLake,
+		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			opts.name = args[0]
```

Comment:
I don't know if it has change but I remember having some weird edge cases where PreRun would ron before validating `cobra.ExactArgs(1),` or before validating flags (or both I don't remember)
So if you can please do q quick pass here that not passing a name will not trigger an index error on the slice would be great

Reply:
Seems to be ok:
```
./bin/mongocli atlas datalake describe
Error: accepts 1 arg(s), received 0
```

Reply:
👍 yep just some some other command doing the same, I just never know which one check before what